olive/passes/onnx/gptq.py

```diff
@@ -0,0 +1,149 @@
+                default_value=128,
+                description="number of samples in calibration dataset to apply quantization. Default value is 128",
+            ),
+            "bits": PassConfigParam(
+                type_=int,
+                default_value=4,
+                description="quantization bits. Default value is 4",
+            ),
+            "dataset": PassConfigParam(
+                type_=str,
+                default_value="wikitext2",
+                description="Calibration dataset. Default value is wikitext2",
+            ),
+            "block_name_to_quantize": PassConfigParam(
+                type_=str,
+                default_value=None,
+                description="Block name to quantize. Default value is model.decoder.layers.",
+            ),
+            "group_size": PassConfigParam(
+                type_=int,
+                default_value=128,
+                description="Block size for quantization. Default value is 128.",
+            ),
+            "batch_size": PassConfigParam(
+                type_=int,
+                default_value=1,
+                description="Batch size for quantization. Default value is 1.",
+            ),
+            "seed": PassConfigParam(
+                type_=int,
+                default_value=0,
+                description="Random seed for sampling calibration dataset. Default value is 0.",
+            ),
+            "damp_percent": PassConfigParam(
+                type_=float,
+                default_value=0.01,
+                description="Damping factor for quantization. Default value is 0.01.",
+            ),
+            "static_groups": PassConfigParam(
+                type_=bool,
+                default_value=False,
+                description="Use static groups for quantization. Default value is False.",
+            ),
+            "true_sequential": PassConfigParam(
+                type_=bool,
+                default_value=False,
+                description="Use true sequential for quantization. Default value is False.",
+            ),
+            "desc_act": PassConfigParam(
+                type_=bool,
+                default_value=False,
+                description="Use descriptive activation for quantization. Default value is False.",
+            ),
+            "sym": PassConfigParam(
+                type_=bool,
+                default_value=False,
+                description="Symmetric quantization. Default value is True.",
+            ),
+            "gpu": PassConfigParam(
+                type_=bool,
+                default_value=True,
+                description="Use GPU for quantization. Default value is True.",
+            ),
+            "export_optimization": PassConfigParam(
+                type_=str,
+                default_value=None,
+                description="Export optimization level. Default value is None.",
+            ),
+        }
+        config.update(get_external_data_config())
```

Comment:
since the model is being saved to file directly using the onnx exporter, these external data configs are not being used.